flink-runtime/src/test/java/org/apache/flink/runtime/jobmaster/JobMasterTest.java

```diff
@@ -1384,8 +1374,9 @@ public void testRequestPartitionState() throws Exception {
                 fail("Expected failure.");
```

Comment:
Let's replace the try-catch block with a `assertThatThrownBy`

Reply:
Fixed